ShippoTesting/TrackTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Shippo;
+
+namespace ShippoTesting
+{
+    [TestFixture ()]
+    public class TrackTest : ShippoTest
+    {
+        [Test ()]
+        public void TestValidGetStatus ()
+        {
+            Shipment shipment = ShipmentTest.getDefaultObject ();
+            Track track = getAPIResource ().RetrieveTracking ("usps", shipment.ObjectId);
```

Comment:
Does this mean the test never worked? If so, could you please avoid requesting a review before completely testing it? My assumption before reviewing is that the code has been completed and tested. Let me know if this is ok with you.